01 Styles/02 Import Bootstrap/README.md

```diff
@@ -64,20 +64,20 @@ used by Bootstrap, we need to set a font appropiated loader:
 ![Demo01_02_Bootstrap.png](../../99 Readme Resources/02 Webpack/Demo01_02_Bootstrap.png "Demo01_02_Bootstrap.png")
 
 - Let's set up the appropiate font / glyphicon loaders. We will install
-first [file-loader](https://github.com/webpack/file-loader). It will produce a new file into our build directory (/dist) and return the
+first [file-loader](https://github.com/webpack/file-loader). It will produce a new file into our build directory (*/dist*) and return the
 public url.
 
 ```
 npm install file-loader --save-dev
 ```
 
-- Producing all the files (images, fonts) in separate files is something we are used to, but what happens if we have a lot small files like this? Initial load of the page would be slower requesting a big bunch of small files, another option that we have is to use [url-loader](https://github.com/webpack/url-loader), in this case the files is encoded as data inside the HTML... and we can combine file-loader / url-loader by adding a condition, encode only files that are smaller that a given size.
+- Producing all the files (images, fonts) in separate files is something we are used to, but what happens if we have a lot small files like this? Initial load of the page would be slower requesting a big bunch of small files, another option that we have is to use [url-loader](https://github.com/webpack/url-loader), in this case the files is encoded as data inside the HTML... and we can combine *file-loader* / *url-loader* by adding a condition, encode only files that are smaller that a given size.
```

Comment:
I'm not sure if *like this* is pointing to file or file**s**. I'll suggest remove it. Any suggestions? :)
I will change these changes later. Thanks @Totemika!.